neuralcoref/neuralcoref.pyx

```diff
@@ -580,7 +581,7 @@ cdef class NeuralCoref(object):
             self.conv_dict.add(key=norm_k, vector=embed_vector/max(len(norm_w), 1))
 
     def __call__(self, doc, greedyness=None, max_dist=None, max_dist_match=None,
-             conv_dict=None, blacklist=None):
+             conv_dict=None, blacklist=None, *, has_mentions=False):
```

Comment:
the meaning of `has_mentions` is unclear. Perhaps `use_existing_mentions` would be better?